packages/venia-ui/lib/components/SearchBar/autocomplete.css

```
@@ -16,15 +16,18 @@
 
 .root {
     background-color: white;
-    border: 1px solid rgb(var(--venia-global-color-border));
-    box-shadow: 0 3px 10px rgba(0, 0, 0, 0.2);
+    border: 2px solid rgb(var(--venia-global-color-gray-600));
+    border-radius: 0 0 6px 6px;
+    border-top-style: none;
+    box-shadow: -6px 6px rgb(var(--venia-brand-color-1-100));
     display: grid;
     font-size: 0.8125rem;
     gap: 0.75rem;
     left: 0;
     padding: 1rem 1rem;
     position: absolute;
     right: 0;
+    top: 2.2rem;
```

Can this be rounded to an integer?

```suggestion
    top: 2.25rem;
```